README.md

```diff
@@ -28,13 +28,13 @@ yarn add react-instascan react instascan-umd
 
 ## Requirements
 To use this library you need at least:
- - react >= 16
+ - react >= 16.3
  - instascan-umd >= 1
 
 # Using it
 
 ## Importing
-React Instascan exposes two Components: **Cameras** and **Scanner**.  
```

Comment:
Can you put the breakline here again?

Reply:
absolutely, my mistake, shouldn't commit all these changes!

Reply:
No worries :)
Thanks!